### DISCUSSION TITLE
p1_simpson_index outcome

### DISCUSSION THREAD

**Discussion by maja-nagler** [IBAHCM/RPiR#559] [Q&A for project] [answered]
Hi,
I am writing to clarify, in what format do You expect p1_simpson_index function to return the Simpson index value in the first demo. The description in the Project 1 suggests that the expected value of Simpson index is:
<img width="354" alt="Screenshot 2023-01-21 at 15 45 10" src="https://user-images.githubusercontent.com/118539738/213874771-b39f3e1d-0faa-495e-8514-46dc2a375b8c.png">

That can be visualised by:
<img width="118" alt="Screenshot 2023-01-21 at 15 23 47" src="https://user-images.githubusercontent.com/118539738/213873797-68e4bcae-0ea5-4ea0-b1db-34620af27dc9.png">

Which in various sources is defined as D2 (or dominance index):
<img width="965" alt="Screenshot 2023-01-21 at 16 07 56" src="https://user-images.githubusercontent.com/118539738/213875746-58ea5c85-a97b-4ce6-b9c8-5099e71686a3.png">

https://search.r-project.org/CRAN/refmans/abdiv/html/simpson.html
https://en.wikipedia.org/wiki/Diversity_index (not saying Wikipedia got it correct)

However according to the second practical and mentioned sources Simpson Index is either 1/D2 (by some source defined as Inversed Simpson Index) or 1-D2:
<img width="661" alt="Screenshot 2023-01-21 at 16 13 35" src="https://user-images.githubusercontent.com/118539738/213875975-5c98a4cf-3d17-496b-9120-1407f964f33b.png">

There is no other mention of the Simpson Index in the first project apart from the one below, which is why I am asking this question.
<img width="373" alt="Screenshot 2023-01-21 at 15 53 12" src="https://user-images.githubusercontent.com/118539738/213875145-2ba18f36-8218-4139-8fad-b55ebc519de0.png">

Are we supposed, according to script return D2 (knowing that it is inconsistent with the following projects), 1-D2 or 1/D2?

I would appreciate clarification, especially in case I was getting it wrong.
 
Kind regards

**Comment by mhuzzell** [comment]
I'm not sure it's necessarily inconsistent with the following projects, as it's sort of arbitrary what the actual values are. I've just been calculating the Simpson index as instructed (whereby smaller values = more biodiversity) and making notes where relevant that the Simpson index is *usually* presented as 1 minus that value.

**Reply by mhuzzell** [reply]
Also worth noting that "how likely two randomly chosen individuals are to be from the same species" does correspond with what you'd expect from an index where smaller = more diverse.

**Reply by maja-nagler** [reply]
I agree.  By consistency I mean phrases like "Run the three functions on the populations provided and check that your results are identical to the original functions" in the following projects (for this, the formula would need to be the same) and the issues of aesthetics, or pedantry associated  with marking (I would want to be sure that I won't be marked down on this).

**Comment by richardreeve** [comment, marked as answer]
The simpson index I'm asking you to calculate throughout is the one I describe in the first project - _"the species proportions from the counts (so the total sums to 1), squares them all individually, and add them all together"_. D2 is 1/Simpson index - The special function is the simpson index, the general function is (well, includes) D2. The "1 minus x" function is nothing I mention in the exercises and nothing I'm interested in.
